index.html

```diff
@@ -1,10 +1,34 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Personal Website - About Me</title>
-    <link rel="stylesheet" type="text/css" href="css/style.css">
-  </head>
-  <body>
-    
-  </body>
-</html>
+<head>
+	<title>Adam Furman Resume</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<img class="one" src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAbbAAAAJDMzNjg1Y2Y2LTBiZDgtNGJiOS1iNWIwLTdiNzc4MGMxM2QzMg.jpg">
+
+<body>
+	<h1>Adam Furman</h1>
+	<ul>
+		<li>
+		<a href="#">About Me</a>
+		</li>
+		<li>
+		<a href="resume.html">Resume</a>
+		</li>
+	</ul>
+  
+<article>
+	<h2>New York</h2>
+	<p>In 2015, I made the move everyone dreams about and came to New York. Currently, I'm an Digital Account Manager for a Media Agency called Zero-In. I specialize in content development, strategy, and project management. I work side by side with International companies and agencies such as: The UN, Equinox, Crunch, Discover, Shake Shack, Melt-shop, Little Beet and more!<br />
+    <br />
```

Comment:
Instead of using these `<br>` tags you should have wrapped each sentence in a `<p>` tag.